File: CS/PS/복습/progr_lv1/2016년.py
def solution(a, b):
    answer = ''
    days = ["FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"]
    m_day = [0,31,29,31,30,31,30,31,31,30,31,30,31]
    total_days = b
    for mon in range(0,a):
        total_days += m_day[mon]
    return days[total_days % 7 - 1]

File: CS/PS/Site/programmers/lv1/2016년.js
function solution(a, b) {
  var answer = '';
  const month = [31,29,31,30,31,30,31,31,30,31,30,31];
  const day = ["FRI","SAT","SUN","MON","TUE","WED","THU"];
  let days = 0;
  for (let mon = 0; mon < a; mon++){
    days += month[mon];
  }
  
  
  return day[(days - (month[a - 1] - b) - 1) % 7];
}

console.log(solution(1,11));